app/src/test/java/com/example/mpweather/ExampleUnitTest.java

```diff
@@ -10,8 +10,5 @@
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
 public class ExampleUnitTest {
-    @Test
-    public void addition_isCorrect() {
-        assertEquals(4, 2 + 2);
-    }
```

Comment:
I took this test out because it seemed like it was not necessary and looked like it was just taking up space.